autopkg/autopkg_tools.py

```diff
@@ -56,6 +57,40 @@ def updated_version(self):
     def name(self):
         return self.plist["Input"]["NAME"]
 
+    def verify_trust_info(self):
```

Comment:
I worry about making `Recipe` a god class. It should be a lightweight object. But this is fine. Vigilant if ever tempted to add more state-mutating methods.